crates/types/src/traits/election.rs

```diff
@@ -58,6 +58,9 @@ pub trait Membership<TYPES: NodeType>:
     /// The members of the committee for view `view_number`.
     fn get_committee(&self, view_number: TYPES::Time) -> BTreeSet<TYPES::SignatureKey>;
 
+    /// Get the networking "topic" pertaining to the committee.
+    fn get_committee_topic(&self) -> Topic;
+
```

Comment:
We will with the Push CDN. Per above, it also looks like I didn't make the change to Libp2p 🙈 , will do that now. The idea is that it will eventually enable the dynamic subscription model that Libp2p and the pCDN both support (which is why we are keeping track in a map which topics we are subscribed to)